pkg/types/alibabacloud/platform.go

```diff
@@ -12,6 +12,27 @@ type Platform struct {
 	// +optional
 	ResourceGroupID string `json:"resourceGroupID"`
 
+	// VpcID is the ID of an already existing VPC where the cluster should be installed.
+	// If empty, a new VPC will created for the cluster.
+	// Destroying the cluster using installer will delete this VPC.
```

Comment:
Why would the cluster destroy delete the VPC? Is that because the VPC must be in the existing resource group? If so, then we are missing some validation. If not, then I don't think that the cluster destroy should delete the existing VPC.

Reply:
I think there is no clear description here. If VpcID is empty, a new VPC will be created, and released when the cluster is destroyed.

Reply:
My suggestion is to remove the language around deleting the VPC.
```
	// VpcID is the ID of the VPC into which the cluster's machines will be created.
	// Leave the VPC ID unset to have the installer create a VPC on your behalf.
```